src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -362,6 +373,13 @@ private void tryToAddVariant(final VariantContext vc, final ReferenceSequence re
                 final String refString = StringUtil.bytesToString(ref, vc.getStart() - 1, vc.getEnd() - vc.getStart() + 1);
 
                 if (!refString.equalsIgnoreCase(allele.getBaseString())) {
+                    // consider that the ref and the alt may have been swapped in a simple biallelic SNP
+                    if (vc.getAlleles().size() == 2 &&
+                            vc.getAlleles().stream().map(Allele::length).allMatch(l -> l == 1) &&
```

Comment:
This is weird.  Why not use isBiallelic() and isSNP()?

Reply:
because I was being too technical and forgot about these utility functions....